orttraining/orttraining/core/graph/gradient_builder.cc

```diff
@@ -1305,6 +1305,14 @@ IMPLEMENT_GRADIENT_BUILDER(GetLayerNormalizationGradient) {
   }
 }
 
+IMPLEMENT_GRADIENT_BUILDER(GetT5LayerNormalizationGradient) {
+  return std::vector<NodeDef>{
+      NodeDef(OpDef{"T5LayerNormalizationGrad", kMSDomain, 1},
+              {GO(0), I(0), I(1), O(2)},
```

Comment:
should be O(1) as the fwd op doesn't have 3rd output

Reply:
Fixed